docs/core/rid-catalog.md

```diff
@@ -26,7 +26,7 @@ RIDs that represent concrete operating systems usually follow this pattern: `[os
 
 - `[version]` is the operating system version in the form of a dot-separated (`.`) version number. For example, `15.10`.
 
-  - The version **shouldn't** be marketing versions, as they often represent multiple discrete versions of the operating system with varying platform API surface area.
+  The version **shouldn't** be a marketing version, as they often represent multiple discrete versions of the operating system with varying platform API surface area.
```

Comment:
Changing versions to singular makes it out of sync with the following "they"

```suggestion
  The version **shouldn't** be a marketing version, as a marketing version often represents multiple discrete versions of the operating system with varying platform API surface area.
```

Reply:
```suggestion
  The version **shouldn't** be a marketing version, as marketing versions often represent multiple discrete versions of the operating system with varying platform API surface area.
```